spec/requests/government_ruleset_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe GovernmentRulesetController, type: :controller do
   describe "GET #new" do
     it "renders the new template" do
+      controller.instance_variable_set(:@ruleset_name, 'government_ruleset')
```

Comment:
There has to be a better way of doing this. 

If the page flow is changed as I recommend (with `ruleset#create` redirecting to either `security_ruleset#new or `government_ruleset#new`) this will become unnecessary.

Reply:
Yes a left over problem from our initial attempt at this. You are correct it is not very restful.